app/components/paperShow/helmet/index.tsx

```diff
@@ -59,8 +63,13 @@ const getStructuredData = (paper: Paper) => {
     image: ['https://assets.pluto.network/scinapse/scinapse-logo.png'],
     datePublished: paper.publishedDate,
     dateModified: paper.publishedDate,
-    about: paper.fosList.map(fos => fos.fos),
+    about: paper.fosList.map(fos => ({ '@type': 'Thing', name: fos.fos })),
     mainEntityOfPage: `https://scinapse.io/papers/${paper.id}`,
+    contentRating: {
+      '@type': 'Rating',
```

Comment:
as I known, it's for 'user' driven rating. also, treating impact factor as rating is very dangerous. 
furthermore, impact factor is for 'journal' not 'paper'.
So, I suggest to delete this block.